config/generated/config.json

```diff
@@ -255,7 +255,7 @@
                                 "properties": {
                                   "value": {
                                     "type": "string",
-                                    "x-go-type-skip-optional-pointer": true,
+                                    "x-go-type-skip-optional-pointer": false,
```

Comment:
These changes look accidental and will change the generated go structs. Please revert changes to config.json